2019/CG-06.md

```diff
@@ -75,7 +82,20 @@ These are a few suggestions for accommodation including the distances to Igalia
     1. Find volunteers for note taking
     1. Adoption of the agenda
     1. Proposals and discussions
-       1. TBD
+       1. [Bulk Memory](https://github.com/WebAssembly/proposals/issues/18) (½ hr)
+         1. Passive element segment encoding
```

Comment:
I'd like to add a second item:

* Possibly simplify runtime conditions for copy direction ([discussion](https://github.com/WebAssembly/bulk-memory-operations/pull/88#issuecomment-492943205)) and for OOB checks